ITR/interfaces.py

```diff
@@ -252,22 +387,25 @@ class PortfolioCompany(BaseModel):
 def pint_ify(x, units='dimensionless'):
     if 'units' in units:
         units = units['units']
-    if x is None or x is np.nan:
+    if x is None or unp.isnan(x):
         return Q_(np.nan, units)
     if type(x) == str:
         if x.startswith('nan '):
             return Q_(np.nan, units)
         return ureg(x)
-    if isinstance(x, Quantity):
+    if isinstance(x, pint.Quantity):
         # Emissions intensities can arrive as dimensionless if emissions_metric and production_metric are both None
-        if x.m is np.nan and x.u == 'dimensionless':
+        if unp.isnan(x.m) and x.u == 'dimensionless':
             return Q_(np.nan, units)
         return x
     return Q_(x, units)
 
 
 def UProjections_to_IProjections(classtype, ul, metric):
-    if ul is None or ul is np.nan:
+    if ul is np.nan or (isinstance(ul, utype) and unp.isnan(ul)):
```

Comment:
does this conditional branch need to be different from line 408 "if ul is None"

Reply:
I have no idea where this code is coming from.  I deleted UProjections_to_IProjections.  I don't see that code here: https://github.com/MichaelTiemannOSC/ITR/blob/template-v2/ITR/interfaces.py

Reply:
indeed, you later remove the class `UProjections_to_IProjections` completely.
I think i commented on an earlier commit, but this line was removed in a later commit.
All right, then, this my comment is resolved